packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -573,6 +577,20 @@ export default class QuanticGeneratedAnswer extends LightningElement {
     return this.state?.expanded;
   }
 
+  /**
+   * Returns whether the component has a custom message to display when no answer is generated.
+   * @returns {boolean}
+   */
+  get hasCustomNoAnswerMessage() {
+    /** @type {HTMLSlotElement} */
+    const slot = this.template.querySelector('slot[name="no-answer-message"]');
+    return !!slot?.assignedNodes()?.length;
+  }
+
+  get shouldDisplayNoAnswerMessage() {
+    return !!this.state?.cannotAnswer && this.hasCustomNoAnswerMessage;
```

Comment:
`cannotAnswer` is already a boolean, so you should not need the `!!`
```suggestion
    return this.state?.cannotAnswer && this.hasCustomNoAnswerMessage;
```